### PR TITLE
Update alert boxes

### PR DESCRIPTION
**Pull request by deadpine**

### CODE REVIEW

File: ui/src/stylesheets/globals/_vars.scss [hunk 1]
```diff
@@ -36,9 +36,10 @@
   --tertiary: #fff;
 
   --color-text: #282846;
+  --color-text-light: #69687d;
```
Comment (frangio) [inline ui/src/stylesheets/globals/_vars.scss]: Can we make this a bit darker? The contrast is too low IMO.

File: ui/src/stylesheets/admonitionblock.scss [hunk 2]
```diff
@@ -8,31 +8,26 @@
 
 .admonitionblock,
 .alert {
-  border: 1px solid rgb(196, 196, 196);
-  border-radius: var(--border-radius);
-  font-family: var(--heading);
-  color: rgb(36, 36, 36);
-  line-height: 100%;
+  display: grid;
```
Comment (frangio) [inline ui/src/stylesheets/admonitionblock.scss]: It's not necessary to use `display: grid` to achieve the desired layout here. The `::before` element should have `display: block` instead.

File: ui/src/stylesheets/admonitionblock.scss [hunk 3]
```diff
@@ -8,31 +8,26 @@
...
-  background-color: white;
+  line-height: 1.5;
+  padding: 1rem .9rem;
   margin: 1.5rem 0;
-  background-repeat: no-repeat;
-  background-size: 32px;
-  background-origin: border-box;
-  background-position: 1.4rem center;
-  
+  word-break: break-word;
```
Comment (frangio) [inline ui/src/stylesheets/admonitionblock.scss]: [According to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#Values), `word-break: break-word` is deprecated, and it is equivalent to `word-break: normal` (the default) combined with `overflow-wrap: break-word`. So we should change this:

```suggestion
  overflow-wrap: break-word;
```

Comment (deadpine) [reply ui/src/stylesheets/admonitionblock.scss]: Sure, thanks Fran

**Review by frangio — APPROVED** [review]
Thanks!

**Pull request merged**